app/src/main/java/de/westnordost/streetcomplete/data/osm/geometry/ElementGeometryTable.kt

```diff
@@ -1,11 +1,11 @@
 package de.westnordost.streetcomplete.data.osm.geometry
 
 object ElementGeometryTable {
-    const val NAME = "elements_geometry"
+    const val NAME_RELATIONS = "elements_geometry_relations"
+    const val NAME_WAYS = "elements_geometry_ways"
```

Comment:
Please give if each table its own `*Table` object to be consistent with the rest of the code

Reply:
Is it ok now?

Reply:
Err no, I meant like all the other tables: Two files, one named WayGeometryTable.kt and the other RelationGeometryTable.kt.
Yes, the columns will be duplicted, but that's okay since they are both two completely independent tables. That the columns have the same names is basically only coincidental.

Reply:
Ok, so the different columns now made me add separate daos as well.

Reply:
Very good, I didn't notice that when scanning over it yesterday. It is good that the Daos are separate because this is consistent with the rest of the code.